src/docs/product/integrations/notification-incidents/slack/index.mdx

```diff
@@ -47,6 +47,12 @@ Your Slack integration is now available to all projects in your Sentry organizat
 
 In the next section, we'll walk you through configuring your notification settings.
 
+### Troubleshooting
```

Comment:
I'd either put this at the bottom of the page or just above the transitional sentence: `In the next section, we'll walk you through configuring your notification settings.`.

Reply:
@mgaeta at the _end_ end or before "Deleting the legacy project-based Slack integration"?

Reply:
@mgaeta also I was following what we do on this page: https://docs.sentry.io/product/integrations/cloud-monitoring/aws-lambda/#troubleshooting